materialize-sql/testdata/validate/migratable-changes.flow.yaml

```diff
@@ -13,9 +13,7 @@ collections:
         dateValue: { type: string }
         datetimeValue: { type: string }
         timeValue: { type: string }
-        dateString: { type: string, format: date }
-        datetimeString: { type: string, format: date-time }
-        timeString: { type: string, format: time }
+        int64: { type: integer, maximum: 9223372036854775808}
```

Comment:
Integer widening is a case I haven't yet added support for in connectors (I tried, it didnt _just work_ for some reason, so I'm going to let it be done in a subsequent PR so we can get this PR merged, it's been sitting for a long time). But added it to the tests so adding it will be easier.